fuzzing/engines/jazzer_launcher.sh

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Google LLC
```

Comment:
2021 for new files (here and everywhere else it applies).

Reply:
> The CI checks also need to be updated, right? I think we should have separate matrices for C++ (libFuzzer / Honggfuzz / etc.) and Java tests (Jazzer / any other engines?).
> 
> Otherwise, the general architecture looks good to me.

Thanks! I create one more Java target that keeps running and added the Java tests as separate matrix tests to the workflow file.